modules/nf-core/yahs/main.nf

```diff
@@ -0,0 +1,44 @@
+process YAHS {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::yahs=1.2a.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/yahs:1.2a.2--h7132678_0':
+        'quay.io/biocontainers/yahs:1.2a.2--h7132678_0' }"
+
+    input:
+    tuple val(meta), path(bed)
+    tuple path(ref), path(fai)
+    val(if_break)
+    val(motif)
+    val(resolutions)
+
+    output:
+    tuple val(meta), path("yahs.out_scaffolds_final.fa"), emit: scaffolds_fasta
```

Comment:
Could you vertically align the `emit:`s for readability? Have a look at other modules to see what I mean.